### ISSUE TITLE
Ignore 'consecutive-duplicates-with-different-values' not applying to duplicate properties within a block

### ISSUE THREAD

**Issue by SKempin** [sasstools/sass-lint#981]
Sass lint reporting 'Duplicate properties are not allowed within a block', despite ignore in config file. 
This warning should not be created?

--  "gulp-sass-lint": "^1.3.2"

-- .yml config file:
```  
  # Disallows
no-duplicate-properties:
    - ignore:
        - consecutive-duplicates-with-different-values
```

-- SASS: 
``` 
 .module .ug-search-links ul {
        float: right;
        font-size: 16px!important;
        font-size: 1.6rem!important;
    }
```

-- Output report: 
697      │ 9        │ warning  │ Duplicate properties are not allowed within a          │ no-duplicate-        ║
║          │          │          │ block                                                  │ properties         

![capture](https://cloud.githubusercontent.com/assets/10877466/21311027/09c6f7fc-c5dd-11e6-8c1c-174130a6839f.JPG)

**Comment by DanPurdy** [comment]
The option you're passing doesn't exist. You need to have a read of the docs for that rule
https://github.com/sasstools/sass-lint/blob/develop/docs/rules/no-duplicate-properties.md

The option you're referencing there exists in stylelint, not sass-lint.
